jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -493,6 +493,16 @@ static void completeInitialization() {
 		throw new InternalError(e.toString());
 	}
 	/*[ENDIF]*/	//!Sidecar19-SE_RAWPLUSJ9&!Sidecar18-SE-OpenJ9
+
+	/*[IF JAVA_SPEC_VERSION >= 19]*/
+	try {
+		java.lang.reflect.Field notifyJvmtiEvents = VirtualThread.class.getDeclaredField("notifyJvmtiEvents"); //$NON-NLS-1$
+		notifyJvmtiEvents.setAccessible(true);
+		notifyJvmtiEvents.set(null, true);
```

Comment:
Can we add this code in an earlier pre-existing `try/catch` block to avoid the new `try/catch` block?

Reply:
I think it's cleaner this way given the `/*[IF !Sidecar19-SE_RAWPLUSJ9&!Sidecar18-SE-OpenJ9]*/`

Reply:
Suggestion: 
```
	try {
		/*[IF !Sidecar19-SE_RAWPLUSJ9&!Sidecar18-SE-OpenJ9]*/
		if (null != systemInitialization) {
			hook = systemInitialization.getMethod("lastChanceHook");	//$NON-NLS-1$
			hook.invoke(null);
		}
		/*[ENDIF]	!Sidecar19-SE_RAWPLUSJ9&!Sidecar18-SE-OpenJ9 */

		/*[IF JAVA_SPEC_VERSION >= 19]*/
 		Field notifyJvmtiEvents = VirtualThread.class.getDeclaredField("notifyJvmtiEvents"); //$NON-NLS-1$
		notifyJvmtiEvents.setAccessible(true);
		notifyJvmtiEvents.set(null, true);
		/*[ENDIF] JAVA_SPEC_VERSION >= 19 */
	} catch (Exception e) {
		throw new InternalError(e.toString());
	}
```